### DISCUSSION TITLE
What about making a new release?

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#782] [General]
More than 2 months without any commits, the CI builds are all deleted. The current official release doesn't include the latest fixes. What about making a new release?

**Comment by LADSoft** [comment]
oops thanks for reminding me, meant to do that back in december...    I'll start a build later tonite (gotta check and make sure everything is still working first) then if that goes well we can have a release when it gets done sometime tomorrow...

**Comment by LADSoft** [comment]
ok i made a new release...   it looks like something is up with occil since the last time I did this though.    Will look into that soon...
